Global/Eclipse.gitignore

```diff
@@ -49,3 +49,15 @@ local.properties
 
 # Code Recommenders
 .recommenders/
+
+# Java class files
+**/*.class
+**/classes
+**/classes/**
+
+# svn plugin
+**/.svn
```

Comment:
We have a precedence here around externalizing other VCSes in their own templates - for example [`Mercurial.gitignore`](https://github.com/github/gitignore/blob/master/Global/Mercurial.gitignore).

We don't have one for Subversion though - do you think that's a suitable compromise here?